sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/dll_import_export.hpp

```diff
@@ -11,33 +11,33 @@
 #pragma once
 
 /**
- * @def AZURE_SECURITY_KEYVAULT_SECRETS_DLLEXPORT
+ * @def AZ_SECURITY_KEYVAULT_SECRETS_DLLEXPORT
  * @brief Applies DLL export attribute, when applicable.
  * @note See https://learn.microsoft.com/cpp/cpp/dllexport-dllimport?view=msvc-160.
  */
 
-#if defined(AZURE_SECURITY_KEYVAULT_SECRETS_DLL) \
-    || (0 /*@AZURE_SECURITY_KEYVAULT_SECRETS_DLL_INSTALLED_AS_PACKAGE@*/)
-#define AZURE_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL 1
+#if defined(AZ_SECURITY_KEYVAULT_SECRETS_DLL) \
+    || (0 /*@AZ_SECURITY_KEYVAULT_SECRETS_DLL_INSTALLED_AS_PACKAGE@*/)
+#define AZ_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL 1
 #else
-#define AZURE_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL 0
+#define AZ_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL 0
 #endif
 
-#if AZURE_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL
+#if AZ_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL
 #if defined(_MSC_VER)
-#if defined(AZURE_SECURITY_KEYVAULT_SECRETS_BEING_BUILT)
-#define AZURE_SECURITY_KEYVAULT_SECRETS_DLLEXPORT __declspec(dllexport)
+#if defined(AZ_SECURITY_KEYVAULT_SECRETS_BEING_BUILT)
+#define AZ_SECURITY_KEYVAULT_SECRETS_DLLEXPORT __declspec(dllexport)
 #else // !defined(AZURE_SECURITY_KEYVAULT_SECRETS_BEING_BUILT)
-#define AZURE_SECURITY_KEYVAULT_SECRETS_DLLEXPORT __declspec(dllimport)
+#define AZ_SECURITY_KEYVAULT_SECRETS_DLLEXPORT __declspec(dllimport)
 #endif // AZURE_SECURITY_KEYVAULT_SECRETS_BEING_BUILT
 #else // !defined(_MSC_VER)
-#define AZURE_SECURITY_KEYVAULT_SECRETS_DLLEXPORT
+#define AZ_SECURITY_KEYVAULT_SECRETS_DLLEXPORT
 #endif // _MSC_VER
 #else // !AZURE_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL
-#define AZURE_SECURITY_KEYVAULT_SECRETS_DLLEXPORT
+#define AZ_SECURITY_KEYVAULT_SECRETS_DLLEXPORT
 #endif // AZURE_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL
 
-#undef AZURE_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL
+#undef AZ_SECURITY_KEYVAULT_SECRETS_BUILT_AS_DLL
 
 /**
  * @brief Azure SDK abstractions.
```

Comment:
Also, below in this file, drop the

```cpp
namespace Azure { namespace Template }
```
with their corresponding comments.
And if you're not updating KV Admin DLLEXPORT macro from `AZURE_` to `AZ_` in this PR, please create another PR which does that, or log a work item. We should fix it before the next KV Admin release.

Reply:
done